README.md

```diff
@@ -1,70 +1,70 @@
-# Getting Started with Create React App
+# Stock Company Information
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+> This is a single page web application that retrieves real live data from the Financial Modeling Prep API. Users can get information on their favourite company at Dow Jones Industrial Avarage.
+> Companies are displayed in the Home Page. Information which inclujde the stock index is displayed at the company information section.
 
-## Available Scripts
+![screenshot](./home.png)
+![screenshot](./company.png)
 
-In the project directory, you can run:
+## Live Demo
```

Comment:
- [ ] Your project is not starting on live demo, kindly update it

Reply:
Updated the live demo by deploying to netlify